protocol/contracts/oracle/src/contract/mod.rs

```diff
@@ -14,7 +13,10 @@ use sdk::{
 use versioning::{package_version, version, SemVer, Version, VersionSegment};
 
 use crate::{
-    api::{Config, ExecuteMsg, InstantiateMsg, MigrateMsg, PriceCurrencies, QueryMsg, SudoMsg},
+    api::{
+        alarms::StableCurrency, Config, ExecuteMsg, InstantiateMsg, MigrateMsg, PriceCurrencies,
```

Comment:
The StableCurrency definition so far has been used for alarms.  With this new use, we say "Hey, the base Oracle currency is the alarm base currency". The reality is the reverse - the alarm base currency -is- the base Oracle currency.